platform/viewer/src/components/UserPreferences/WindowLevelPreferences.js

```diff
@@ -1,19 +1,115 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import { redux } from '@ohif/core';
 
-import { TabFooter } from '@ohif/ui';
+import { TabFooter, useSnackbarContext } from '@ohif/ui';
 import { useTranslation } from 'react-i18next';
 
+const { actions } = redux;
```

Comment:
We can probably nuke the `redux` dependency if this broadcasts save/restore and hotkeys manager handles localstorage persistence? Using redux is probably fine for this PR, but I'm not sure hoisting state that high has much value

Reply:
To nuke redux dependency we should update hotkeysManager so we could store hotkeys together with options (such as window level preferences). This sould be worked by this issue: https://github.com/OHIF/Viewers/issues/1447